contrib/cmake/vcpkg_extra_triplets/x64-windows-mega.cmake

```diff
@@ -9,6 +9,7 @@ set(VCPKG_CRT_LINKAGE dynamic)
 if(PORT MATCHES "ffmpeg" OR 
    PORT MATCHES "openssl" OR 
    PORT MATCHES "curl" OR 
+   PORT MATCHES "sdl1" OR 
```

Comment:
It would be nice to have one fewer DLLs when we can, I see that SDL 2 was licensed under the zlib licence (and specifically to get away from LGPL probably)- I don't suppose we could update to SDL2 ?